deploy/dev-ngsa-asb-eastus/monitoring/grafana/05-grafana.yaml

```diff
@@ -13,6 +13,8 @@ spec:
 apiVersion: apps/v1
 kind: Deployment
 metadata:
+  annotations:
+    checkov.io/skip1: CKV_K8S_35=Grafana container requires env var for secret value
```

Comment:
This rule is definitely specific for Grafana. However, is it worth adding the rule exception in the workflow where the check is running, rather than updating all these grafana yaml files? Only issue with that approach is that the rule will be applied to all deployment files. Thoughts @joaquinrz?